.github/workflows/regression-tests.yml

```diff
@@ -19,25 +30,35 @@ jobs:
         uses: actions/checkout@v4
         with:
           repository: 0xPolygon/kurtosis-cdk
-          ref: main
+          ref: "v0.2.15"
```

Comment:
Why not use the `v0.2.22`?
https://github.com/0xPolygon/kurtosis-cdk/releases/tag/v0.2.22